Integrations/CentralBankIntegrationLib/XmlSerializationObjects/ValCurse.cs

```diff
@@ -13,6 +13,6 @@ public class ValCurs
         [XmlElement("Valute")]
         public List<Valute> Valutes { get; set; }
 
-        public List<string> TopValutesCharCode => new List<string> { "EUR", "USD", "GBP", "CNY", "JPY" };
+        public List<string> TopValutesCharCode => new List<string> {  };
```

Comment:
Should we initialize the property at all? If so, then why with =>?

Reply:
Right now it is the way to define top currencies. Sorry, don't understand what do you suggest?